modules/puppetserver/manifests/init.pp

```diff
@@ -140,10 +140,10 @@
         ],
     }
 
-    file { '/home/puppet-users':
+    file { '/home/letsencrypt-admins':
```

Comment:
We can rather absent this, and do the ensure for the letsencrypt-admins home directory in the LetsEncrypt module, since it is for that, if this were to ever change also (although highly unlikely) it would need moved anyways, so for consistency maybe this should be in the LetsEncrypt module?